include/libdnf/base/transaction.hpp

```diff
@@ -33,17 +33,20 @@ along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
 
 namespace libdnf::base {
 
-
+/// Error related to processing transaction
 class TransactionError : public Error {
 public:
     using Error::Error;
+    /// @return Return a domain of the Error Class
     const char * get_domain_name() const noexcept override { return "libdnf::base"; }
+    /// @return Return a name of the Error
     const char * get_name() const noexcept override { return "TransactionError"; }
```

Comment:
I would rephrase it as "Return Error Class's domain name" and "Return Error Class' name"

Reply:
I just realized I corrected slightly wrong. It should be “class'” not “class's”. My mistake. Since all other corrections are good, I pushed this one character correction and I will merge it now.